src/Modules/Lifecycle.php

```diff
@@ -23,7 +23,8 @@ class Lifecycle {
     'swiftmailer',
     'tracker',
     'block_place',
-    'aggregator'
+    'aggregator',
+    'event_log_track_ui'
```

Comment:
```suggestion
    'event_log_track_ui',
```
PHP lets us add a comma on the last item of an array which is handy for future changes so you don't have to touch that line when adding a new one.